src/uct/tcp/tcp.h

```diff
@@ -166,16 +170,18 @@ typedef struct uct_tcp_cm_state {
  * TCP Connection Manager event
  */
 typedef enum uct_tcp_cm_conn_event {
+    /* Acknowledgment for the packet with magic number. */
+    UCT_TCP_CM_CONN_MAGIC_NUMBER_ACK  = UCS_BIT(0),
```

Comment:
it looks a bit odd that sending ack is a connection event, while sending number itself is not

Reply:
added